luaui/Widgets/gui_com_nametags.lua

```diff
@@ -144,23 +144,38 @@ local function GetCommAttributes(unitID, unitDefID)
 	local playerRank
 	local name = ''
 	local luaAI = Spring.GetTeamLuaAI(team)
+	Spring.Echo("IsDecoy:", UnitDefs[unitDefID].customParams.decoyfor)
 	if luaAI and luaAI ~= "" and string.find(luaAI, 'Scavengers')  then
-
+		--name = "Scav Commander" -- todo: i18n this thing
+		if UnitDefs[unitDefID].customParams.decoyfor then
+			name = Spring.I18N('units.scavdecoycommandernametag', {})
```

Comment:
I18N without any interpolated data don't need to be passed an empty table.
`name = Spring.I18N('units.scavdecoycommandernametag')

Reply:
> I18N without any interpolated data don't need to be passed an empty table. `name = Spring.I18N('units.scavdecoycommandernametag')`

Leftover from me trying to get this thing to work, will fix if I don't forget.